### ISSUE TITLE
Cannot resolve module 'child_process' to build XMLHttpRequest

### ISSUE THREAD

**Issue by aksonov** [webpack/webpack-dev-server#66]
<code>
ERROR in ./~/parse/~/xmlhttprequest/lib/XMLHttpRequest.js
Module not found: Error: Cannot resolve module 'child_process' in /Users/aksonov/UPDT/sf/node_modules/parse/node_modules/xmlhttprequest/lib
 @ ./~/parse/~/xmlhttprequest/lib/XMLHttpRequest.js 15:12-36

</code>

**Comment by ShuaiYuan87** [comment]
Hi @jhnns I tried to use an alias for the xmlhttprequest module. The way I did is I created a dummy a.js file and I put module.exports = XMLHttpRequest; in it. In my make-webpack-config.js I have 
var alias = {
        xmlhttprequest$: "./a.js"
    };
Now when I start production server I have error 
module.exports = XMLHttpRequest;
	                 ^
ReferenceError: XMLHttpRequest is not defined
It seems webpack cannot find XMLHttpRequest (the original one supposedly is supported by the browser?) Is there something I'm missing?

Thanks!
